docs/evaluate/development-production-features/multi-team.mdx

```diff
@@ -0,0 +1,94 @@
+- **Enhance security by isolating different parts of the application**, reducing the risk and impact of potential issues.
+- **Continue using familiar tools like queue-based workers**, without the need for complex new setups or deployments.
+
+## Core Components of Temporal Nexus
+
+Temporal Nexus supports improved modularity and multi-team collaboration, by allowing teams to define a clean Nexus Service API contract for other teams to use, that abstract underlying Temporal primitives, like Workflows.
+
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus API Registry, along with service documentation that enable others to use the Nexus Services provided by a Nexus Endpoint. This enables teams to compose Nexus Services into their workflows, both within and across namespace boundaries.
+
+Nexus is designed to connect durable executions across team and namespace boundaries with an integrated Temporal SDK experience, built-in Nexus Machinery that processes Nexus RPC calls on your behalf, and promotes a more modular architecture for sharing a subset of your team’s capabilities via a well-defined Nexus Service API contract.
+
+**Nexus API Registry:** A `Nexus API Registry` is a list of Nexus `Endpoints` available within a Temporal environment. 
+Adding a Nexus `Endpoint` allows Nexus Operation requests to be sent to the Worker listening on the Endpoint’s target Namespace and Task Queue, from a caller Workflow using the name of the Endpoint in the Nexus API Registry.
+
+**Nexus Endpoints:** A Nexus `Endpoint` is like an API proxy that can serve one or more Nexus `Services` from a `Worker` listening on a target Namespace and Task Queue. 
+This enables presenting a clean API contract and hiding the underlying Temporal implementation, which may consist of internal `Workflows`, `Signals`, and other Temporal primitives – or arbitrary code. 
+In Temporal Cloud an endpoint’s access-control policy determines which callers are allowed to access a Nexus `Endpoint`.
+
+**Nexus Services:** A Nexus `Service` is a named collection of arbitrary-duration Nexus `Operations` that provide an API contract suitable for sharing across team boundaries. 
+Nexus Services are registered with a `Worker` that is listening on the target Namespace and Task Queue for an `Endpoint`.
+
+**Nexus Operations:** A Nexus `Operation` is not intrinsically durable, but can be made durable by backing it with Temporal primitives like a `Workflow`, `Update`, or `Signal`.
+The Temporal SDK provides helper functions to create Nexus Operations that may be registered with a Nexus Service in a `Worker`. 
+Synchronous Nexus `Operations` can also execute arbitrary code, but unlike `Activities` they have a short request timeout (under 10 seconds) and don’t support [heartbeats](https://community.temporal.io/t/best-practices-for-long-running-activities/934).
```

Comment:
We mention Synchronous here. Should we mention Asynchronous explicitly vs just referencing Temporal primitive?

Reply:
looks like this has been fixed, will take another